### PR TITLE
MAINT: bring back np.lib tests

### PR DESCRIPTION
**Pull request by jaimefrio**
Merging #5476 removed automatic test discovery for np.lib, this PR adds back numpy/lib/setup.py to fix that.

### CODE REVIEW

File: numpy/lib/setup.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+def configuration(parent_package='',top_path=None):
+    from numpy.distutils.misc_util import Configuration
+
+    config = Configuration('lib', parent_package, top_path)
+    config.add_data_dir('tests')
+    return config
+
+if __name__ == '__main__':
+    from numpy.distutils.core import setup
+    setup(configuration=configuration)
```
Comment (charris) [inline numpy/lib/setup.py]: Any idea what the marker at the line ending is? Carriage return linefeed?

Comment (jaimefrio) [inline numpy/lib/setup.py]: If you hover over it it complains about "No newline at end fo file", let me fix that.

Comment (charris) [inline numpy/lib/setup.py]: Live and learn...

**Comment by charris** [comment]
Thanks Jaime.

**Pull request merged**